### PR TITLE
Show plymouth debug messages during boot

### PR DESCRIPTION
**Pull request by kravciak**
This shoud provide more information to resolve bsc#995310

Local run: http://dhcp91.suse.cz/tests/3065
Needles: https://gitlab.suse.de/openqa/os-autoinst-needles-sles/merge_requests/249

### CODE REVIEW

File: tests/installation/grub_test.pm [hunk 1]
```diff
@@ -80,9 +73,26 @@ sub run() {
         }
         send_key 'ret';
     }
-    if (get_var("XEN")) {
+    elsif (get_var("XEN")) {
```
Comment (okurz) [inline tests/installation/grub_test.pm]: I don't see this as related. Should go into individual commit

File: tests/installation/grub_test.pm [hunk 2]
```diff
@@ -80,9 +73,26 @@ sub run() {
...
         send_key_until_needlematch("bootmenu-xen-kernel", 'down', 10, 5);
     }
+    elsif (check_var('ARCH', 'aarch64') && check_var('DISTRI', 'sle') && get_var('PLYMOUTH_DEBUG')) {
+        send_key 'e';
+        # Move to end of kernel boot parameters line
+        send_key_until_needlematch "linux-line", "down";
+        send_key "end";
+        if (check_screen("linux-line-matched")) {
+            # remove "splash=silent quiet showopts"
+            for (1 .. 28) { send_key "backspace" }
```
Comment (okurz) [inline tests/installation/grub_test.pm]: magic numbers are evil

File: tests/installation/grub_test.pm [hunk 3]
```diff
@@ -80,9 +73,26 @@ sub run() {
...
+            type_string 'plymouth:debug';
+            record_soft_failure "Running with plymouth:debug to catch bsc#995310";
+        }
+        else {
+            record_soft_failure "Unable to match linux line";
```
Comment (okurz) [inline tests/installation/grub_test.pm]: That's not a proper soft failure. I suggest to die instead

File: tests/installation/grub_test.pm [hunk 4]
```diff
@@ -8,15 +8,8 @@
...
-#    Livecdreboot test name was unclear, renamed it in to install_and_reboot.
-#    The code concerning grub test has moved to new test grub_test.pm
-#    Main pm adapted for the new grub_test.pm
-#    In first_boot.pm added get_var(boot_into_snapshot) for assert linux-terminal,
-#    since after booting on snapshot, only a terminal interface is given, not GUI.
-#
-#    Issues on progress: 9716,10286,10164
-# G-Maintainer: dmaiocchi <dmaiocchi@suse.com>
+# Summary: Handle grub menu after reboot
+# Maintainer: Martin Kravec <mkravec@suse.com>
```
Comment (okurz) [inline tests/installation/grub_test.pm]: Can you add '# Tags: poo#9716, poo#10286, poo#10164' ?

File: tests/installation/grub_test.pm (hunk 2 above)
Comment (kravciak) [inline tests/installation/grub_test.pm]: That's why I check_screen - to make sure I can use those magic numbers (better suggestion?)

**Pull request merged**